docs/concepts/cloning.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Ondat Cloning"
+linkTitle: "Ondat Cloning"
+weight: 1
+---
+
+## Overview
+
+> 💡 This feature is available in release `v2.10.0` or greater.
+
+The Ondat Cloning feature adds support for specifying existing PVCs in the dataSource field to indicate a user would like to clone a Volume.
+
+### What is a Clone?
+
+A **clone** is defined as a duplicate of an existing Kubernetes Volume that can be consumed as any standard Volume would be.
+
+### Why use Cloning:
+
+TODO.
```

Comment:
Someone from CS may have some concrete use cases for here?

Reply:
Cloning is very useful when you need to copy the data from one persistent volume to another. For example there could be an application error where an issue is caused by a data quality issue. Being able to copy the contents of a volume and debug this without affecting the running application can be extremely useful when attempting to fix such issues.